### PR TITLE
FIX: Check whether config.json changes

### PR DESCRIPTION
**Pull request by LaurenSpiegel**
Before overwriting config.json, confirm changes
have been made with env variables.

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -94,8 +94,10 @@ if [[ "$RECORDLOG_ENABLED" ]]; then
     JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .recordLog.enabled=true"
 fi
 
+if [[ JQ_FILTERS_CONFIG != "." ]]; then
```
Comment (jonathan-gramain) [inline docker-entrypoint.sh]: shouldn't it be `$JQ_FILTERS_CONFIG`?

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -94,8 +94,10 @@ if [[ "$RECORDLOG_ENABLED" ]]; then
...
 jq "$JQ_FILTERS_CONFIG" config.json > config.json.tmp
```
Comment (jonathan-gramain) [inline docker-entrypoint.sh]: minor: indent this block

**Comment by electrachong** [comment]
Are they supposed to be unable to mount the config.json file if they don't specify any env variables?

An example from our [docs](http://s3-server.readthedocs.io/en/latest/DOCKER/#specifying-your-own-host-name):
`docker run -v $(pwd)/config.json:/usr/src/app/config.json -p 8000:8000 -d scality/s3server`

**Comment by LaurenSpiegel** [comment]
thanks, @jonathan-gramain, updated.

**Pull request merged**